tests/validate/04-builtin-suites.t

```diff
@@ -39,9 +39,17 @@ with open(file_name, 'r') as in_file:
 set_test_number $((( ((${#SUITES[@]})) * 2 )))
 #-------------------------------------------------------------------------------
 # Validate suites.
+NO_EMPY=true
+if python -c 'import em' 1>'/dev/null' 2>&1; then
```

Comment:
This test is not sufficient - see https://github.com/cylc/cylc/pull/2734#issuecomment-409001677 and preceding comments.  
@TomekTrzeciak did it this way (after modifying `cylc check-software` to do a more comprehensive test):
https://github.com/cylc/cylc/blob/master/tests/empy/00-simple.t#L20

Reply:
Fair enough. Check updated.